tests/functional/TestSmokeTest/tests/block_volume_test.go

```diff
@@ -0,0 +1,73 @@
+// +build functional
+
+//
+// Copyright (c) 2015 The heketi Authors
```

Comment:
Need to update the date here as this is a new file.

Reply:
done